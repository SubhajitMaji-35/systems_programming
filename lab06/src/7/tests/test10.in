1 0 0 0
0 1 0 0
0 0 1 0
2 4 6 8
1 3 5 7
9 11 13 15
