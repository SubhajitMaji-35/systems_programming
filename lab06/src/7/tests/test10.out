3 4 6 8
1 4 5 7
9 11 14 15
